app/scripts/modules/google/serverGroup/details/scalingPolicy/scalingPolicy.directive.html

```diff
@@ -1,9 +1,11 @@
 <dl ng-class="InsightFilterStateModel.filtersExpanded ? '' : 'dl-horizontal dl-wide'" style="margin-bottom: 20px">
-  <dt>
-    Target {{policy.basis}}
-    <help-field key="{{policy.basisHelpKey}}"></help-field>
-  </dt>
-  <dd ng-repeat="target in policy.targets">{{target}}</dd>
+  <ul ng-repeat="basis in policy.bases">
+    <dt>
+      Target {{basis.description}}
+      <help-field key="{{basis.helpKey}}"></help-field>
+    </dt>
+    <dd ng-repeat="target in basis.targets">{{target}}</dd>
+  </ul>
```

Comment:
Probably shouldn't mix `<ul>`s with `<dd/dt>`s - although I'm sure the browser renders it fine, it's not really "correct" or "valid" markup. Browsers let you get away with murder, though.

If you change this to:
```html
<dt ng-repeat-start="basis in policy.bases">
  Target {{basis.description}}
</dt>
<dd ng-repeat-end>
  <div ng-repeat="target in basis.targets">{{target}}</div>
</dd>
```
does that still render as you'd expect?